src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -29,50 +28,76 @@ object WdlExpression {
 
   val preEvaluableFunctions_SingleParameter: Seq[String] = Seq("read_int", "read_string")
```

Comment:
constants in objects should have the first letter capitalized. also can you get unaryOperators while you're there?